GitTfs/Commands/Fetch.cs

```diff
@@ -146,6 +146,9 @@ private void FetchRemote(bool stopOnFailMergeCommit, IGitTfsRemote remote)
 
         protected virtual void DoFetch(IGitTfsRemote remote, bool stopOnFailMergeCommit)
         {
+            if (upToChangeSet > 0 && InitialChangeset.HasValue && InitialChangeset.Value > upToChangeSet)
```

Comment:
The other code checks for `x == -1` or `x != -1`. Can this change to check for that, too? It looks like `0` will not get caught by this check, but be processed as a [desired stop version](https://github.com/cmakshin/git-tfs/blob/3a7894bc6b4009fec7545168eefb804b1afed30d/GitTfs/Core/GitTfsRemote.cs#L346-L347).

As an aside, I'm not thrilled about using `-1` like this, as a magic number, even though I probably had a hand in doing that initially. Do you think it would make sense to make it a nullable int and `null` instead of `-1`? Or maybe add an extension method that encapsulates the check for `-1` vs not?